pkg/reconciler/instances/connectivityproxy/action.go

```diff
@@ -35,7 +50,7 @@ func (a *CustomAction) Run(context *service.ActionContext) error {
 		}
 	}
 
-	if binding != nil && app == nil {
+	if (binding != nil && app == nil) || (binding != nil && app != nil) {
```

Comment:
Should we install it even if there is no binding? I thought that it should be installed only when user creates a binding for the Connectivity Service

Reply:
Oh, my bad :d It's fine. I got lost in the logical operands